FF144 MediaDevices.getUserMedia() method allows resizeMode constraint (#41152)
mdn/content: files/en-us/mozilla/firefox/releases/144/index.md

- {{domxref("RTCDataChannel")}} instances are now [transferrable objects](/en-US/docs/Web/API/Web_Workers_API/Transferable_objects), and hence can be passed to [workers](/en-US/docs/Web/API/Worker). ([Firefox bug 1209163](https://bugzil.la/1209163)).
- The [`closing` event](/en-US/docs/Web/API/RTCDataChannel/closing_event) and the `onclosing()` event handler are now supported on the {{domxref("RTCDataChannel")}} interface. ([Firefox bug 1611953](https://bugzil.la/1611953)).
- The {{domxref("MediaDevices/getUserMedia","getUserMedia()")}} and {{domxref("MediaDevices/getDisplayMedia","getDisplayMedia()")}} methods of the {{domxref("MediaDevices")}} interface now support the [`resizeMode`](/en-US/docs/Web/API/MediaTrackConstraints#resizemode) constraint.

>   This constraint allows developers to request that the browser return video that matches other constraints, such as resolution and frame rate, even if the requested stream is not supported by the underlying hardware.
>   The browser may then crop, downscale resolution, and reduce the frame rate, of video captured from the camera, or downscale (but not crop) video captured from a screen or window. ([Firefox bug 1286945](https://bugzil.la/1286945)).

Reviewer: - "request the browser to return video" -> can probably be shortened to "request video"
```suggestion
  This constraint allows developers to request video that matches other constraints, such as resolution and frame rate, even if the requested constraints are not supported by the underlying hardware.
  The browser may then crop, downscale, or reduce the frame rate of the video captured from a camera, or downscale (but not crop) the video captured from a screen or window. ([Firefox bug 1286945](https://bugzil.la/1286945)).
```
